src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -683,4 +704,149 @@ public function column_cb( $post ) {
 			<div class="locked-indicator"></div>
 		<?php
 	}
+
+	/**
+	 * Displays the pagination.
+	 *
+	 * @since 5.2.2
+	 * @access protected
+	 *
+	 * @param string $which
+	 */
+	protected function pagination( $which ) {
+		if ( empty( $this->_pagination_args ) ) {
+			return;
+		}
+
+		$total_items     = $this->_pagination_args['total_items'];
+		$total_pages     = $this->_pagination_args['total_pages'];
+		$infinite_scroll = false;
+		if ( isset( $this->_pagination_args['infinite_scroll'] ) ) {
+			$infinite_scroll = $this->_pagination_args['infinite_scroll'];
+		}
+
+		if ( 'top' === $which && $total_pages > 1 ) {
+			$this->screen->render_screen_reader_content( 'heading_pagination' );
+		}
+
+		$output = '<span class="displaying-num">' . sprintf(
+			/* translators: %s: Number of items. */
+			_n( '%s item', '%s items', $total_items ),
+			number_format_i18n( $total_items )
+		) . '</span>';
+
+		$current              = $this->get_pagenum();
+		$removable_query_args = wp_removable_query_args();
+
+		$current_url = set_url_scheme( 'http://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'] );
+
+		$current_url = remove_query_arg( $removable_query_args, $current_url );
+
+		if( isset( $_REQUEST['s'] ) ) {
+			$current_url = add_query_arg( 's', $_REQUEST['s'], $current_url );
```

Comment:
* :warning: Processing form data without nonce verification.
* :warning: Detected usage of a non-sanitized input variable: $_REQUEST['s']

<sub>hash: af2570c415c159436dfaf3fad79781b8</sub>